lms/djangoapps/certificates/signals.py

```diff
@@ -66,7 +66,7 @@ def _listen_for_id_verification_status_changed(sender, user, **kwargs):  # pylin
     grade_factory = CourseGradeFactory()
     expected_verification_status, _ = SoftwareSecurePhotoVerification.user_status(user)
     for enrollment in user_enrollments:
-        if grade_factory.read(user=user, course=enrollment.course_overview).passed:
+        if grade_factory.read(user=user, course_key=enrollment.course_overview.id).passed:
```

Comment:
Why was this change necessary?

Reply:
@thallada This is because course grade_factory `read` method expects `course` not `course_overview` object. It will get the course from course_key. Also, for determining `passed` status, `compute_passed` method requires course `grade_cutoffs` attribute and course overview does not have grade_cutoffs. 

https://github.com/edx/edx-platform/blob/ac935185894a336f68cffb8a0b3856f45598ee8b/lms/djangoapps/grades/course_grade_factory.py#L16-L41

https://github.com/edx/edx-platform/blob/b5435e6f5a785ba81f7cf1d339fdccf90acc683b/lms/djangoapps/grades/course_data.py#L67-L71

https://github.com/edx/edx-platform/blob/b6a5d5f40efd424c028535127e89cfc54450b0ec/lms/djangoapps/grades/course_grade.py#L245